perception/navigator_vision/nodes/database_color.py

```diff
@@ -0,0 +1,351 @@
+#!/usr/bin/env python
```

Comment:
This is great!

It would be nice if you moved the ImageHolder and Debug to navigator_tools so others could use them.

But not necessary!